### PR TITLE
Move cmd state test to pytest

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Move cmd state test to pytest

### CODE REVIEW

File: tests/pytests/unit/states/test_cmd.py [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    Test to download a script and execute it, specifically on windows.
+    Make sure that when using runas, and a password is not supplied we check.
+    """
+    name = "cmd.script"
+    ret = {
+        "name": "cmd.script",
+        "changes": {},
+        "result": False,
+        "comment": "",
+        "commnd": "Must supply a password if runas argument is used on Windows.",
```
[the remark this answers is not in the document]
Comment (krionbsd) [reply tests/pytests/unit/states/test_cmd.py]: Yep, fixed

File: tests/pytests/unit/states/test_cmd.py [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+# 'call' function tests: 1
+
+
+def test_call():
+    """
+    Test to invoke a pre-defined Python function with arguments
+    specified in the state declaration.
+    """
+    name = "cmd.script"
+    #         func = 'myfunc'
```
[the remark this answers is not in the document]
Comment (krionbsd) [reply tests/pytests/unit/states/test_cmd.py]: Fixed

File: salt/states/cmd.py [hunk 3]
```diff
@@ -1048,7 +1048,7 @@ def script(
         return ret
 
     if runas and salt.utils.platform.is_windows() and not password:
-        ret["commnd"] = "Must supply a password if runas argument is used on Windows."
+        ret["command"] = "Must supply a password if runas argument is used on Windows."
```
Comment (twangboy) [inline salt/states/cmd.py:1051]: I would suspect this is supposed to be `comment` if it's part of the state return. I don't know what the state does with additional keys... they may be ignored

**Review by twangboy — CHANGES_REQUESTED** [review]

**Review by twangboy — DISMISSED** [review]

**Comment by s0undt3ch** [comment]
@Ch3LL, @twangboy : Do we need a changelog entry for the typo fixed?

**Comment by Ch3LL** [comment]
Yeah, I think it would be good since it changes something for the user.

**Review by Ch3LL — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**